mono/metadata/mempool.c

```diff
@@ -124,6 +136,18 @@ mono_mempool_new_size (int initial_size)
  *
  * Free all memory associated with this pool.
  */
+#if defined(__has_feature)
+#if __has_feature(thread_sanitizer)
+// clang's ThreadSanitizer detects races of total_bytes_allocated and pool->d.allocated throughout the functions
+//   * mono_mempool_alloc
+//   * mono_mempool_new_size
+//   * mono_mempool_destroy
+// while these races could lead to wrong values, total_bytes_allocated is just used for debugging / reporting and since
+// the mempool.c functions are called quite often, a discussion led the the conclusion of ignoring these races:
+// https://bugzilla.xamarin.com/show_bug.cgi?id=57936
+__attribute__ ((no_sanitize("thread")))
+#endif
+#endif
```

Comment:
Should I prefix the macro with `MONO_` (making it `MONO_NO_SANITIZE_THREAD`) like all the other `__attribute__` macros in that file?